class UserNotExistError(Exception):
    pass


class DuplicateUserError(Exception):
    pass
